### PR TITLE
FLEX-5532_Fixed platform pom

### PR DESCRIPTION
**Pull request by joostknapen**

### CODE REVIEW

File: osgp/platform/osgp-adapter-kafka-distributionautomation/pom.xml [hunk 1]
```diff
@@ -11,19 +11,21 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <artifactId>osgp-adapter-kafka-distributionautomation</artifactId>
```
Comment (smvdheijden) [inline osgp/platform/osgp-adapter-kafka-distributionautomation/pom.xml:14]: Undo this reordering. See also sonar rule "_pom elements should be in the recommended order (java:S3423)_":

_The POM Code Convention is the Maven project's internal recommendation for POM element ordering. It calls for listing modifiers in the following order:_

```
<modelVersion/>
<parent/>
<groupId/>
<artifactId/>
<version/>
<packaging/>
<name/>
<description/>
<url/>
<inceptionYear/>
<organization/>
<licenses/>
<developers/>
<contributors/>
<mailingLists/>
<prerequisites/>
<modules/>
<scm/>
<issueManagement/>
<ciManagement/>
<distributionManagement/>
<properties/>
<dependencyManagement/>
<dependencies/>
<repositories/>
<pluginRepositories/>
<build/>
<reporting/>
<profiles/>
```

**Review by smvdheijden — CHANGES_REQUESTED** [review]

File: osgp/platform/osgp-adapter-kafka-distributionautomation/pom.xml (hunk 1 above)
Comment (joostknapen) [reply osgp/platform/osgp-adapter-kafka-distributionautomation/pom.xml:14]: Hmm, I  just made this in line with all other modules. I'll revert this change

**Comment by smvdheijden** [comment]
Retest this please.

File: osgp/platform/pom.xml [hunk 2]
```diff
@@ -115,6 +115,10 @@
     <module>osgp-adapter-domain-tariffswitching</module>
     <module>osgp-adapter-domain-smartmetering</module>
 
+    <!-- kafka -->
```
Comment (smvdheijden) [inline osgp/platform/pom.xml:118]: If you add these modules, you should probably have to add these two dependencies to components in shared  as well:
```
    <module>../shared/osgp-kafka-distributionautomation</module>
    <module>../shared/osgp-kafka-logger</module>
```

**Review by smvdheijden — CHANGES_REQUESTED** [review]

File: osgp/platform/pom.xml (hunk 2 above)
Comment (joostknapen) [reply osgp/platform/pom.xml:118]: These modules are already in the pom

**Comment by smvdheijden** [comment]
Retest this please.

File: osgp/platform/pom.xml (hunk 2 above)
Comment (smvdheijden) [reply osgp/platform/pom.xml:118]: Are you sure, I do not see them in this pom?

Comment (joostknapen) [reply osgp/platform/pom.xml:120]: Oh, sorry, I was looking at the osgp-applications repo. They where added in that location. I just added them in this file.

**Review by smvdheijden — APPROVED** [review]

**Pull request merged**